app/models/user/communication_preferences.rb

```diff
@@ -22,6 +22,7 @@ def self.product_keys
       email_on_nudge_notification
       email_about_fundraising_campaigns
       email_about_events
+      email_on_onboarding
```

Comment:
Can you rename this to `receive_onboarding_emails` pls? As it's not an "on X" style notification.